app/templates/summary.html

```diff
@@ -43,7 +43,7 @@ <h2 class="summary__title saturn" id="{{summary_id}}">{{block.title|safe}}</h2>
           <div class="summary__answer">
             {% set item_id = id ~ "-" ~ loop.index0 %}
             <div class="summary__answer-text" id="{{item_id}}-answer" data-qa="answer-{{outer_loop_index}}-{{inner_loop_index}} answer-type-{{item.type}}">
-              {%- if sub_item.answer is none -%}
+              {%- if not sub_item.answer -%}
```

Comment:
Changed this back the way it was. In Jinja templating it should be none as opposed to None. Rather than the NoneType it refers to a test definition in the Jinja templating code. See https://github.com/pallets/jinja/issues/520